config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -706,7 +706,7 @@ presubmits:
           - --deployment=node
           - --gcp-zone=us-west1-b
           - --node-args=--image-config-file=/home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/containerd/image-config-serial.yaml
-          - '--node-test-args=--service-feature-gates=AllAlpha=true --feature-gates=SidecarContainers=true --container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/usr/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd*\"]}"'
+          - '--node-test-args=--feature-gates=AllAlpha=true,EventedPLEG=false --service-feature-gates=AllAlpha=true,StorageVersionAPI=false,APIServerIdentity=false,StorageVersionMigrator=false,UnknownVersionInteroperabilityProxy=false,MutatingAdmissionPolicy=false --container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/usr/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd*\"]}"'
```

Comment:
Is there a corresponding periodic for this?

Reply:
It was suggested at https://github.com/kubernetes/test-infra/pull/30281#pullrequestreview-1594875513. However, it doesn't seem to have been added.

It looks like a job with the similar name was deleted a long time ago by #18974.